maat-court-data-api/src/main/java/gov/uk/courtdata/prosecutionconcluded/helper/ProcessSentencingHelper.java

```diff
@@ -0,0 +1,46 @@
+package gov.uk.courtdata.prosecutionconcluded.helper;
+
+import gov.uk.courtdata.repository.CrownCourtProcessingRepository;
+import gov.uk.courtdata.util.DateUtil;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+
+import static gov.uk.courtdata.enums.CrownCourtCaseType.APPEAL_CC;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class ProcessSentencingHelper {
+
```

Comment:
This is not a helper, this is an implementation class. Please follow the pattern with the other flows.